src/Ens/JobeetBundle/Controller/JobController.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Ens\JobeetBundle\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+use Ens\JobeetBundle\Entity\Job;
+use Ens\JobeetBundle\Form\JobType;
+
+/**
+ * Job controller.
+ *
+ */
+class JobController extends Controller
+{
+
+    /**
+     * Lists all Job entities.
+     *
```

Comment:
@PhotonSapiens 
please keep an eye on not adding extra lines in docblocks
in this specific case you don't need docblocks for controller methods at all